tests/conftest.py

```diff
@@ -3121,8 +3121,8 @@ def fds_data_context(
     datasource.add_query_asset(
         name="trip_asset_partition_by_event_type",
         query="SELECT * FROM table_partitioned_by_date_column__A",
-    ).add_partitioner_column_value("event_type")
-
+    )
+    # add_partitioner_column_value("event_type")
```

Comment:
Should this be updated to add an actual partitioner?

Reply:
removed in [43888ed](https://github.com/great-expectations/great_expectations/pull/9499/commits/43888edfeef513cbc3abbcb2d26ab1cb34fc9fa5)